spyder/plugins/workingdirectory/plugin.py

```diff
@@ -113,7 +108,21 @@ def __init__(self, parent, workdir=None, **kwds):
                                       _('Change to parent directory'),
                                       triggered=self.parent_directory)
         self.toolbar.addAction(parent_action)
-                
+
+    def get_workdir(self):
+        """Get current workdir from the CONF file."""
+        if self.get_option('startup/use_fixed_directory'):
+            workdir = self.get_option('startup/fixed_directory',
+                                      default='')

```

Comment:
If you use `default` here, there's no need to add `'startup/fixed_directory'` to `config/main.py`

Reply:
this option is not included in `config/main.py`